style.css

```diff
@@ -0,0 +1,136 @@
+body {
+  font-family: poppins, sans-serif;
+}
+
+.parent-container {
+  display: flex;
+  position: absolute;
+  height: 718px;
+  width: 375px;
+}
```

Comment:
[OPTIONAL] Just as the previous review suggested, it will be good if we could center your parent-container, I think you can achieve it by giving a dynamic value to the parent-container eg: `width: 100%` instead of a fixed one, I hope you don't mind verifying that it really works?